portfolio/src/types.ts

```diff
@@ -0,0 +1 @@
+export type ExperienceLevel = "Beginner" | "Intermediate" | "Advanced";
```

Comment:
I went ahead and created a new file for you to put all of your `type` and `interface` definitions into, specifically types and interfaces that need to be shared and used in different places.